caravel/assets/javascripts/SqlLab/components/ResultSet.jsx

```diff
@@ -43,7 +46,7 @@ class ResultSet extends React.Component {
                 >
                   <i className="fa fa-line-chart m-l-1" /> Visualize
                 </Button>
-                <Button bsSize="small">
+                <Button bsSize="small" onClick={this.csv.bind(this)}>
```

Comment:
i think this makes more sense as a simple `<a>` tag since the `csv()` is just setting window.location.

```
const url = `/caravel/csv/${this.props.query.id}`;
<a className="btn btn-sm" href={`/caravel/csv/${url}`}">
  ...
</a>
```